demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml

```diff
@@ -325,6 +325,15 @@ parameters:
     saml_sp_x509_string: ''
     saml_registration_url: ''
 
+    # knpu_oauth2_client settings
```

Comment:
You need to define oauth_keycloak_client_id, oauth_keycloak_client_secret, oauth_keycloak_auth_server_url and oauth_keycloak_realm, not the ones defined here